buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/PerformanceTest.java

```diff
@@ -18,34 +18,90 @@
 
 import org.gradle.api.internal.tasks.options.Option;
```

Comment:
I guess we should use `org.gradle.api.tasks.options.Option` instead of this deprecated internal API.

Reply:
Nice catch!